tests/scancode/data/apache-1.1.txt

```diff
@@ -0,0 +1,58 @@
+/* ====================================================================

```

Comment:
Please avoid to put this file at the root of the data dir...  instead create a subdirectory named after the module/thing you are testing